tools/clang/test/hlsl_cmds_test/dump_dependency.test

```diff
@@ -0,0 +1,24 @@
+// RUN: %dxc /T ps_6_0 %s /M | FileCheck %s
+
+// RUN: %dxc /T ps_6_0 %s /MFdeps | FileCheck --input-file=%S/deps %s
```

Comment:
In lit `%T` resolves to a temporary directory and `%t` to a temporary file path. Both get cleaned up by lit.